### ISSUE TITLE
cannot delete pad anymore via API

### ISSUE THREAD

**Issue by mrtopf** [ether/etherpad-lite#3638]
We have a problem some time now not being able to delete pads via API. What happens is usually, that the api call times out and then etherpad restarts.

Unfortunately the pad is still there afterwards in different conditions. Sometimes all the content is deleted but the timeslider is still there with stars. Sometimes the whole content stays. 

I also don't see anything in the logfile, only that node takes up 100% cpu.

We are running:

- MariaDB 10.1.40 (unfortunately still with MyISAM). Also the DB is 94G 
- Node v8.16.0
- git version of etherpad

Is there anything I can do to debug that a bit more? 
(and how would I delete a pad manually in the database? So I can e.g. check if it's a database problem)

BTW, this used to work. 

Thanks in advance!

**Comment by muxator** [comment]
Sorry for the delay.

I just deleted a pad on a local instance and on https://beta.etherpad.org with this call:

```
curl "http://localhost:9001/api/1/deletePad?apikey=<APIKEY>&padID=<PAD_NAME>"
```

* Etherpad: current development version 695c2d2e84b3 (which is a bit after **1.8.0-beta1**)
* node **10.x**

I have made a fix about pad deletion in 40408134470bdfca3175147b64b7ad4348dd9c9a, but that happened on March 26, and your issue is from months later. Anyway, is you are on the stable version (==on master) that fix landed in **1.8.0-beta.1**.

Could you please confirm which version you are running now, and if the problem is still there?

Thanks

**Comment by muxator** [comment]
Closing this, deletion via API works in Etherpad >= **1.8.0**.
